build.sh

```diff
@@ -57,6 +57,24 @@ do
   esac
 done
 
+if [ "$DO_UPDATE" == true ] && [ "$QUIET" == false ]; then
+  echo "Checking if html-build is up to date..."
+  ORIGIN_URL=$(git remote get-url origin)
+  git fetch -q $ORIGIN_URL master
```

Comment:
I think -q should only be if $QUIET is set. It seems important to know what is happening to my local git checkout.